app/Actions/GithubApp/CreateGithubAppFromManifest.php

```
@@ -20,12 +21,21 @@ public function create(string $code): GithubApp
+                'body' => $response->body(),
+            ]);
+
+            $message = (string) ($response->json('message') ?? '');
+
             throw ValidationException::withMessages([
-                'github_app' => __('Failed to convert manifest (HTTP :status).', [
+                'github_app' => __('Failed to convert manifest (HTTP :status):status_message.', [
                     'status' => $response->status(),
+                    'status_message' => $message !== '' ? ' '.$message : '',
                 ]),
             ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\ValidationException' in method 'create'. (undefined)

(StaticAccess)